include/evmc/helpers.h

```diff
@@ -0,0 +1,52 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+/**
+ * EVMC Helpers
+ *
+ * A collection of helper functions for invoking a VM instance methods.
+ * These are convenient e.g. for Go bindings, because this is the only way to invoke a function
```

Comment:
Would reword that to something like: `These are convenient for languages where invoking foreign runtime loaded functions is "ugly" or problematic (such as Go).`